README.md

```diff
@@ -1,69 +1,70 @@
-| network_management_default_gateway      | string                       |                          | Add a default route via this gateway address                                                                                                       |     N    |
-| network_management_pre_up               | string                       | `""`                     | Commands to execute before any other action is performed                                                                                           |     N    |
-| network_management_post_up              | string                       | `""`                     | Commands to execute after all other network operations are performed                                                                               |     N    |
-| network_management_nameservers          | list of strings              | `['8.8.8.8', '8.8.4.4']` | List of all nameservers to use                                                                                                                     |     N    |
-| network_management_clear_bridges        | boolean                      | `False`                  | Remove all ovs bridges before recreate them, useful for renaming of bridges                                                                        |     N    |
-| network_management_reboot_for_config    | boolean                      | `False`                  | Reboot target server to setup new network config, useful for major network configuration, which may require manual interactions otherwise          |     N    |
-| interfaces                              | [list of dicts](#interfaces) | `[]`                     | List of all interfaces to setup, keep in mind it can cause various errors if you configure a interface here and later use it as a port on a bridge |     N    |
-| bridges                                 | [list of dicts](#bridges)    | `[]`                     | List of network bridges to setup (all bridges are managed by openvswitch)                                                                          |     N    |
+| Option                                    | Type                         | Default                  | Description                                                                                                                                        | Required |
+|-------------------------------------------|------------------------------|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|:--------:|
+| `network_management_always_script`        | boolean                      | `False`                  | Generate a systemd service and network management script always (overwrites networking.service if exist)                                           |     N    |
```

Comment:
"Always generate a networking script instead of a `/etc/network/interfaces` config, not only when the system is not running Debian."